### ISSUE TITLE
API and machinery for managing switch link TX equalization parameters.

### ISSUE THREAD

**Issue by rcgoodfellow** [oxidecomputer/omicron#6554]
The default switch TX equalization settings we ship with Dendrite do not work for all transceivers. In some cases, it is necessary to change this parameter for a particular transceiver. The omicron API should provide a way for operators to configure TX equalization at RSS time and dynamically through the API.

- [ ] RSS per-link TX equalization setting
- [ ] API per-link TX equalization setting

Dendrite already provides [API endpoints](https://github.com/oxidecomputer/dendrite/blob/76c735d472e3badaeca08982e22496fccb1ce210/dpd/src/tofino_api_server.rs#L271-L309) to manage TX equalization.

**Comment by bnaecker** [comment]
@Nieuwejaar looks like this was resolved in https://github.com/oxidecomputer/omicron/pull/6914, should we close this?

**Comment by Nieuwejaar** [comment]
> [@Nieuwejaar](https://github.com/Nieuwejaar) looks like this was resolved in [#6914](https://github.com/oxidecomputer/omicron/pull/6914), should we close this?

Nope.  This is the issue that keeps coming up in the network sync meetings.  This is all plumbed through to omicron, but omicron itself doesn't yet export an interface that will allow an admin to modify the settings.
